.github/workflows/e2e.yml

```diff
@@ -18,12 +18,12 @@ jobs:
 
       # the soroban CLI & RPC source code to compile and run from system test
       # refers to checked out source of current git hub ref context
-      SYSTEM_TEST_SOROBAN_CLI_REF: ${{ github.workspace }}
-      SYSTEM_TEST_SOROBAN_RPC_REF: https://github.com/stellar/soroban-rpc.git
+      SYSTEM_TEST_SOROBAN_CLI_REF: ${{ github.workspace }}/soroban-cli
+      SYSTEM_TEST_SOROBAN_RPC_REF: https://github.com/stellar/soroban-rpc.git#main
```

Comment:
I'm not sure on this ref to main, probably will want to pin this to last stable release tag?

Reply:
yeah I can set it to the right tags and merge the PR! no need to wait for it to pass since main is the same as the tags right now imo

Reply:
did it!